### ISSUE TITLE
Outdated Kotlin Gradle plugin version

### ISSUE THREAD

**Issue by iamcosmin** [alexrintt/shared-storage#124]
**Describe the bug**
When building the app for android, this error comes up:
```
FAILURE: Build failed with an exception.

* What went wrong:
The Android Gradle plugin supports only Kotlin Gradle plugin version 1.5.20 and higher.
The following dependencies do not satisfy the required version:
project ':shared_storage' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.50
```

**To Reproduce**
1. Upgrade Android Gradle Plugin to 7.3.0.
2. Run `flutter build apk`

**Expected behavior**
The app should build successfully, without any error.

**Comment by alexrintt** [comment]
Hi! Thanks for reporting.

Please, try it:

```yaml
dependencies:
  # ...
  shared_storage:
    git:
      url: https://github.com/alexrintt/shared-storage
      ref: 124
```

And see if that works.

---

# Changes:

- Upgraded Android Gradle Plugin to `7.4.2`.
- Upgraded Gradle to `7.5`.
- Upgraded `shared_storage` and `shared_storage/example` Kotlin version to `1.8.21`.

**Comment by alexrintt** [comment]
Published as v0.8.0
